ophyd/device.py

```diff
@@ -1205,7 +1205,14 @@ def _instantiate_component(self, attr):
             # Initial access of signal
             cpt = self._sig_attrs[attr]
         except KeyError:
-            raise AttributeError(f'Component does not exist: {attr}') from None
+            raise RuntimeError(
+                f'Component {attr!r} exists at the Python level and '
+                'the has triggered the `_instantiate_component` '
```

Comment:
Missing a word here, can't quite tell which one.

Reply:
there should be no first "the" in the second line?

Reply:
Not that, still wouldn't make sense.  Perhaps the missing word is _Device_? As in 

> Component {attr!r} exists at the Python level and the Device has triggered the `_instantiate_component`

Reply:
or 

> ... the caller has triggered ...

Reply:
@tacaswell would you like to fix up the second half-sentence? thanks, will pass the review immediately once it is the way you want it (and looks like a complete sentence)

Reply:
Y'all knew what I meant...